SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/EventTracingMaps/EventTracingCustomGDKFilterMap.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/EventTracingMaps/EventTracingCustomGDKFilterMap.h"
+
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTestGameMode.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/SettingsTests/CustomGDKFilterTest.h"
+
+UEventTracingCustomGDKFilterMap::UEventTracingCustomGDKFilterMap()
+	: UGeneratedTestMap(EMapCategory::CI_NIGHTLY_SPATIAL_ONLY, TEXT("EventTracingCustomGDKFilterMap"))
+{
+}
+
+void UEventTracingCustomGDKFilterMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+
+	AddActorToLevel<ACustomGDKFilterTest>(CurrentLevel, FTransform::Identity);
+
+	ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
+	WorldSettings->DefaultGameMode = AEventTracingTestGameMode::StaticClass();
+	WorldSettings->SettingsOverride = {
+		"../../../Engine/Plugins/UnrealGDK/SpatialGDK/Config/MapSettingsOverrides/TestOverridesCustomGDKFilter.ini"
```

Comment:
Are these files already in the GDK? I can't see them.
Is there a reason why `UGeneratedMap.SetCustomConfig` is insufficient?

Reply:
Does this work if using a project plugin vs engine plugin?

Reply:
Ahh I think I need to change the git ignore. Will look at SetCustomConfig though...

Reply:
Looks good!

Reply:
You changed my world Miron.